acceptance-tests/tests/commands/create.spec.ts

```diff
@@ -1,8 +1,7 @@
-import { ENTER } from '../lib/prompt';
-
 import { describe, beforeAll, afterAll, it, expect } from 'vitest';
 import rimraf from 'rimraf';
-import { TestState } from '../lib/testState';
+import { ENTER } from '../../lib/prompt';
+import { TestState } from '../../lib/TestState';
```

Comment:
The file name is `testState`, so this will work on macos since it's not case sensitive, but will fail on other OS

Reply:
Ah right, I committed the file casing change which should fix this